spark-api-jupyter/templates/service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  {{- if .Values.service.loadBalancerIP }}
```

Comment:
Is it admissible that we use ephemeral IPs for this? Maybe this IP should be mandatory?

Reply:
There are a number of configs that can be done, especially since ingress is not removed. That's why I wanted to cover the general case for the moment